cl/scrapers/tasks.py

```diff
@@ -70,14 +71,17 @@ def update_document_from_text(
             opinion.cluster.__dict__.update(data)
         elif model_name == "Citation":
             data["cluster_id"] = opinion.cluster_id
-            ModelClass.objects.get_or_create(**data)
+            _, citation_created = ModelClass.objects.get_or_create(**data)
+            metadata_dict["Citation"]["created"] = citation_created
```

Comment:
Is it possible that the update will fail on OpinionCluster or Docket or is citation the only one where it could fail to create something.

Reply:
Also - if we run this code - and we have a mistake in our regex pattern 
we could create citations with no known reporter - I wonder if we should validate that our citations are actually citations.

Reply:
You are right we should validate `extract_from_text`  output before ingesting. 

We have 3 options for this:

1. change all juriscraper `Site.extract_from_text` to return citation strings to be parsed using `eyecite.find_citations`. This would unify the ingestion process between `extract_from_text` and the scrapers, but demands a lot of changes on juriscraper

2. validate the returned Citation objects, just checking that the reporter exists in `reporters-db` and that the reporter matches the Citation type returned

3. implement (2.) and also enable ingesting citation strings from `extract_from_text`, so we can start migrating the scrapers to (1.)


I have implemented (2.) for this PR